report/renderer/details-renderer.js

```diff
@@ -61,12 +61,13 @@ export class DetailsRenderer {
       case 'screenshot':
       case 'debugdata':
       case 'full-page-screenshot':
+      case 'entity-classification':
       case 'treemap-data':
         return null;
 
       default: {
-        // @ts-expect-error tsc thinks this is unreachable, but be forward compatible
-        // with new unexpected detail types.
+        // @ts-expect-error - all `details.type`s need to be handled above so tsc thinks this is unreachable.
```

Comment:
only because this is just rewording an existing comment for clarity, will I super nit pick like this but; I find the mixing of "Code" and english kinda awkward and wish this would just read as `all detail types ...`.

up to you tho

Reply:
agree, i'll change it. :)